src/snapshotStateHandler.js

```diff
@@ -0,0 +1,24 @@
+import { SnapshotState } from "jest-snapshot";
+
+let states = {};
+
+module.exports = {
+    create: ({snapshotPath, ci, update}) => {
```

Comment:
The `ci` and `update` arguments can stay internal here (hardcoded to the process environment variables); the public interface should only take the path

Reply:
Good idea! :)